resources/java_memory_assistant/bin/clean-up.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -e
```

Comment:
I think in the Cloud Foundry community Go is the language of choice these days, for targeted utilities like this.